draft-ietf-masque-connect-ip.md

```diff
@@ -1438,6 +1438,10 @@ There are initially two entries in this registry:
 |      ip      |  IP Proxying | This Document |
 {: #iana-suffixes-table title="New MASQUE URI Suffixes"}
 
+Designated experts for this registry are advised that they should approve all
+requests unless they believe the requested Path Segment will conflict with
+future IETF work.
```

Comment:
```suggestion
Designated experts for this registry are advised that they should approve
requests that meet the following criteria:
- the request has sufficient documentation and explanation to be implementable
- the use case is relevant to proxying
- the requested Path Segment will not conflict with existing or expected future IETF work
```

Reply:
@DavidSchinazi how about this?

Reply:
I'm removing the first bullet because this is Expert Review not Specification Required. Otherwise this sounds good.

```suggestion
Designated experts for this registry are advised that they should approve
requests that meet the following criteria:
- the use case is relevant to proxying
- the requested Path Segment will not conflict with existing or expected future IETF work
```